src/contrib/mlir/lowerer.cpp

```diff
@@ -136,8 +137,10 @@ namespace
     void DialectLoweringPass::populateNGraphToAffineConversionPatterns(
         OwningRewritePatternList& patterns)
     {
-        RewriteListBuilder<NGAddOpConversion, NGDotOpConversion, NGReturnOpConversion>::build(
-            patterns, &getContext(), *this);
+        RewriteListBuilder<NGAddOpConversion,
+                           NGReluOpConversion,
+                           NGDotOpConversion,
+                           NGReturnOpConversion>::build(patterns, &getContext(), *this);
```

Comment:
Since this is going to grow a lot, maybe better follow alphabetic order? Same for visitors below?

Reply:
I will instead use an .inc file. Let me see.